lib/sportradar/api/nfl/stat.rb

```diff
@@ -0,0 +1,7 @@
+module Sportradar
```

Comment:
It's just there for namespacing, see the full class name for the individual stat classes. I wanted to separate these stat classes from the top level NFL namespace.